### ISSUE TITLE
  ** (EXIT) time out on Github Actions step

### ISSUE THREAD

**Issue by churcho** [kimlai/tz_world#18]
I am getting the following error in my Github Action step for testing my app

```
exited in: GenServer.call(TzWorld.Backend.DetsWithIndexCache, {:timezone_at, %Geo.Point{coordinates: {-64.13932679999999, 41.0775432}, properties: %{}, srid: nil}}, 10000)
         ** (EXIT) time out
```        

What could be the cause and what options are there to mitigate this?

**Comment by kipcole9** [comment]
@churcho,.any chance you can post here a more complete log? I run the test suite for this library in GitHub actions without issues and just ran the lookup in your example of my incredibly slow MacBook without issue (with no timezone found for those coordinates BTW - is that what you expect? Its in the Southern Ocean between South Africa and Antartica?). I don't include timezone data for locations that aren't on land to reduce the amount of data.

TLDR; Surprised to see a GenServer timeout and hoping a more complete log will give me some clues. Any chance this is a public repo so I can try to reproduce?

**Comment by churcho** [comment]
Sorry I fiddled with the return vars. I will try and recreate this in a minimal setup and tag you.
